packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript

```diff
@@ -1,8 +1,8 @@
 name="brave-browser-nightly-deb"
-version="1.37.44"
+version="1.38.6"
 gives="brave-browser-nightly"
 breaks="${gives}-bin ${gives}-app"
 url="https://github.com/brave/brave-browser/releases/download/v${version}/${gives}_${version}_amd64.deb"
 description="The Brave browser is a fast, private and secure web browser for PC, Mac and mobile. This is the nightly build of Brave Browser"
-hash="4b656207fab91977a59a529a5ff2502c5e9f5969838174f11c4dbfeff46e37b8"
+hash="da93a1b5c7febb45686979927a4bc81e878bf144ee676c16e4e9d6f621e4a85f "
 maintainer="Korkmatik <9c54efc1-143b-4f5f-9d2f-f6f189583f0f@aleeas.com>"
```

Comment:
Replace Korkmatik's details with yours @theblobscp . If you're willing to maintain it of course.